docs/infrastructure-components.md

```diff
@@ -0,0 +1,164 @@
+# Infrastructure components
+
+We describe here the elements that form the infrastructure required for projects
+under KubeVirt's umbrella. This mainly involves components required to run our
+CI/CD systems and related services, like the monitoring stack or applications for
+indexing and querying build logs.
+
+The CI/CD system is based on [Prow], kubernetes CI system. For a brief overview
+of how Prow runs jobs take a look at ["Life of a Prow Job"]. To see common Prow
+usage and interactions flow, see the pull request interactions [sequence diagram].
+
+## Layout
+
+![infra-layout](infra-layout.png)
+
+## Clusters
+
+Our infrastructure currently uses 3 main clusters. For each of them we will
+include the following fields:
+
+* Description: brief explanation of the cluster purpose.
+* Context: context name of the cluster in the master kubeconfig used by Prow to
+access it. It is also the value of the `cluster` field in the Prow jobs that run
+on the cluster.
+* Components: which workloads run in the cluster, according to the layout picture
+above.
+* External services: functionality not running in our infra that is required for
+the normal operation of the workloads running in the cluster.
+* Connected clusters: for each cluster, which others are required for its normal
+operation.
+* Exposed services: functionality accessible from outside the cluster, usually as
+web applications.
+
+### Control plane
+
+#### Description
```

Comment:
to me, the "Description" section is pointless. I'd just have the text follow the name of the cluster

Reply:
Sure done